### PR TITLE
Update database

### LINKED ISSUE #2: Database modification
### Necessary modifications in the database:

* Rename scheme to **evo_exchange**

* Add in the table **users**:
    - [x] relationship _one-to-many_ to table **phones**
    - [x] relationship _one-to-many_ to table **user_photos**
    - [x] relationship _one-to-many_ to table **children**

* Create new tables with relationship _many-to-one_ to table **users**:
    - [x]  **phones**(_id, user_id, phone_number_)
    - [x]  **user_photos**(_id, user_id, photo_)
    - [x]  **children**(_id, user_id, age, sex_)

* Add in the table **advertisements**: 
    - [x]  fields: _category, description, product_condition, age, sex(we can use enum: girl, boy, unisex)
    - [x]  relationship _one-to-many_ to table **advertisements_images**
    
*  Create new table with relationship _many-to-one_ to table **advertisements**:
    - [x] **advertisements_images**(_id, advertisement_id_, image)

* Add in tables **addresses** and **locations** field _district_,
    - [x]  The field _city_ in these tables should be enam with the biggest cities and towns of Ukraine and district centers of Kharkiv region

### PR DESCRIPTION
**Pull request by sergej-kozionov**
Fixes #2 

### Proposed Changes
Alters database tables.

### CODE REVIEW

**Comment by sergej-kozionov** [comment]
@Jack11M take a look, please.

File: src/main/java/com/hillel/evoApp/model/Sex.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.hillel.evoApp.model;
+
+public enum Sex {
+    GIRL, BOY
```
Comment (Jack11M) [inline src/main/java/com/hillel/evoApp/model/Sex.java]: I suggest to add GENDER_NEUTRAL or UNISEX

File: src/main/java/com/hillel/evoApp/model/Sex.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.hillel.evoApp.model;
+
+public enum Sex {
```
Comment (Jack11M) [inline src/main/java/com/hillel/evoApp/model/Sex.java]: We can rename to Gender

File: src/main/java/com/hillel/evoApp/model/Child.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @Column(name = "age")
+    private Long age;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "sex")
+    private Sex sex;
```
Comment (Jack11M) [inline src/main/java/com/hillel/evoApp/model/Child.java]: We can rename to Gender

File: src/main/resources/liquibase/db.changelog-1.1.xml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+            <column name="user_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="age" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="sex" type="VARCHAR(30)">
```
Comment (Jack11M) [inline src/main/resources/liquibase/db.changelog-1.1.xml]: We can rename to Gender

File: src/main/resources/liquibase/db.changelog-1.1.xml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true" unique="true" nullable="false"/>
+            </column>
+
+            <column name="user_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="age" type="BIGINT">
+                <constraints nullable="false"/>
```
Comment (Jack11M) [inline src/main/resources/liquibase/db.changelog-1.1.xml]: Maybe it is better to remove constraints nullable="false", if user do not want to write age of his/her child

File: src/main/java/com/hillel/evoApp/model/Age.java [hunk 6]
```diff
@@ -0,0 +1,5 @@
+package com.hillel.evoApp.model;
+
+public enum Age {
+    LESS_THAN_1_YEAR, YEAR_1
```
Comment (Jack11M) [inline src/main/java/com/hillel/evoApp/model/Age.java]: Maybe it is better to create 
class Age 
with fields:
int number
TypeOfTime tipeOfTime

Enum TypeOfTime{
YEARS, MONTHS
}

Comment (sergej-kozionov) [reply src/main/java/com/hillel/evoApp/model/Age.java]: Shouldn't we store the number of days as long? and convert it to years months?
or store just a date, that will be converted on our side to days/months?

Comment (Jack11M) [reply src/main/java/com/hillel/evoApp/model/Age.java]: I like idea with Date of Birth and convert it to months, if age less than 1 or 2, and to years in another case.

Comment (sergej-kozionov) [reply src/main/java/com/hillel/evoApp/model/Age.java]: I think date of birth is not the thing the user will want to share, but we can predict it to weeks/months if the UI form will have fields like: `X` - Years, `Y` - Months, then we can convert them to date in the past and use it like an approximate date of birth

File: src/main/resources/liquibase/db.changelog-1.1.xml (hunk 5 above)
Comment (sergej-kozionov) [reply src/main/resources/liquibase/db.changelog-1.1.xml]: Thanks

File: src/main/java/com/hillel/evoApp/model/City.java [hunk 7]
```diff
@@ -0,0 +1,5 @@
+package com.hillel.evoApp.model;
+
+public enum City {
+    KYIV, KHARKIV
```
[the remark this answers is not in the document]
Comment (sergej-kozionov) [reply src/main/java/com/hillel/evoApp/model/City.java]: Lets use capabilities of the `enum` and solve its integration with the database.

[the remark this answers is not in the document]
Comment (sergej-kozionov) [reply src/main/java/com/hillel/evoApp/model/City.java]: I have consulted with Dmitrii and we came to the idea that database should keep only English names, and we use static storage to introduce the localization. You can find it in any Spring Boot localization/internationalization tutorials.

**Pull request merged**